docs/guides/design.md

```diff
@@ -0,0 +1,281 @@
+  {: .no}
+- Put the most important parameters first, and use [optional parameters][parameters_optional] when a value is not strictly required.
+  {: .yes}
+
+<section class="tutorial-row" markdown>
+<div markdown>
+!!! failure "Don't"
+    ```
+    AddTask(project, task, labels, reccurence,
+      workflow, dueDate)
+    ```
+</div>
+<div markdown>
+!!! success "Do"
+    ```
+    AddTask(task, project, [dueDate])
+    ```
+</div>
+</section>
+
+
+### Use simple names
+
+When building a Pack you don’t need to worry about name collisions, and accessibility to users is more important than completeness or accuracy. When choosing a name, prefer simple nouns or verbs and remove any extraneous detail.
+
+- Don’t include the Pack or company name.
+  {: .no}
+- Avoid unnecessary detail in names, unless required to distinguish them.
+  {: .no}
+- Use names that sound more like ordinary speech.
+  {: .yes}
+- Prefer single nouns or verbs when feasible.
+  {: .yes}
+
+<section class="tutorial-row" markdown>
+<div markdown>
+!!! failure "Don't"
+    ```
+    AcmeTasksListAllTasks()
+    AcmeTasksCreateFromScannedImageUpload()
+    AcmeTasksSetAssignee()
+    ```
+</div>
+<div markdown>
+!!! success "Do"
+    ```
+    Tasks()
+    AddFromPhoto()
+    Reassign()
+    ```
+</div>
+</section>
+
+
+## API Integration
+
+A common use case for Packs is integrating with another application or service using their API. While each integration is unique, there are certain patterns and conventions that can be useful to understand. This section includes some tips for designing a Pack around an existing API.
+
+
+### Select collections
+
+Most REST APIs are organized into collections, usually corresponding specific types of items in the application. An API can contain dozens of collections, but as per the general guidance above it's best to start with the handful of core ones that are most valuable to users.
+
+??? example "Example: Todoist"
+
+    The [Todoist API][todoist_reference] includes collections for Projects, Sections, Tasks, Comments, and Labels. While a power user may want to leverage all of that information, for most users Projects and Tasks are the core entities they'll want to work with.
+
+
+### Design the schema
+
+Examine the data returned for each item in the collection and determine what to expose to users in Coda. Select the fields most important to users and start there. You can always add more fields later without breaking anything.
+
+When designing your schema, select user-friendly names for your properties. The field in the API may use technical terminology or refer to an older name no longer in use by the product.
+
+??? example "Example: Todoist task schema"
+
+    The Todoist API returns [up to 20 fields][todoist_task] for a task, but for most use cases only a few are required. Additionally the name "content" is replaced with "name", and the field "id" renamed to "taskId".
```

Comment:
The `[up to 20 fields][todoist_task]` doesn't seem to render as an actual link

![Screen Shot 2023-03-01 at 11 08 34 AM](https://user-images.githubusercontent.com/99229954/222240272-4a8c6846-7740-4562-a240-b5cef114da44.png)

Reply:
Good catch! I removed the link, since the example response below is illustrative enough.